datalad/support/annexrepo.py

```diff
@@ -12,23 +12,53 @@
 
 """
 
-from os.path import join, exists
+from os import getcwd
+from os.path import join as p_join, exists, normpath, isabs, commonprefix, relpath
```

Comment:
we should unify how we import this path.join... I did use pjoin at some point, simple join at other spots, then converged to use "opj" as testkraut uses... p_join becomes fourth one I believe.  IMHO -- should be either simple join or opj -- which one?  I would vote now for a simple join -- it is unlikely to collide with anything.